main/ZactuatorFASTLED.ino

```diff
@@ -0,0 +1,293 @@
+/*
+  OpenMQTTGateway - ESP8266 or Arduino program for home automation
+
+  This gateway enables to use RGBLED strips like WS2812
+
+    Copyright: (c)
+
+    This file is part of OpenMQTTGateway.
+
+    OpenMQTTGateway is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    OpenMQTTGateway is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "User_config.h"
+
+#ifdef ZactuatorFASTLED
+
+#ifdef ESP8266
+#include <FastLED.h>
+#else
+#include <FastLED.h>
+#endif
+#include <ArduinoJson.h>
+enum LEDState
+{
+  OFF,
+  FIRE,
+  GENERAL
+};
+LEDState currentLEDState;
+long lastUpdate = 0;
+long currentUpdate = 0;
+CRGB leds[FASTLED_NUM_LEDS];
+CRGB ledColorBlink[FASTLED_NUM_LEDS];
+StaticJsonBuffer<200> jsonBuffer;
+bool blinkLED[FASTLED_NUM_LEDS];
+const long blinkInterval = 300;
+const long fireUpdate = 10;
```

Comment:
@1technophile should this be defined as preprocessor macro?

Reply:
I see that "jsonBuffer" is not used any more. I remove it and commit...